python/lsst/fgcmcal/fgcmBuildStars.py

```diff
@@ -189,8 +189,8 @@ def fgcmMakeAllStarObservations(self, groupedDataRefs, visitCat,
         # want to store checkpoint files.  If both are set, we will
         # do checkpoint files.  And if only one is set, this is potentially
         # unintentional and we will warn.
-        if (visitCatDataRef is not None and starObsDataRef is None or
-           visitCatDataRef is None and starObsDataRef is not None):
+        if (visitCatDataRef is not None and starObsDataRef is None
+           or visitCatDataRef is None and starObsDataRef is not None):
```

Comment:
I would normally have indented this second line more, but if flake8 likes it, I guess that's fine.

Reply:
Yes, but four character indents cause alignment problems with the first line inside the block so things are always a bit confused. I end up moving it around until flake8 stops complaining and that's the indent that it already had.